tests/integration/features/manifests_int_test.go

```diff
@@ -123,4 +124,29 @@ metadata:
 		Expect(realNs.Name).To(Equal("real-file-test-ns"))
 	})
 
+	It("should set the managed annotation when feature is managed", func() {
```

Comment:
Not sure we need such a test scenario. It focuses on the "how" part (implementation detail) whereas we should be testing the behavior (if it's actually managed). 

I was thinking - how about converting the entire `resources_int_test.go` test cases to be feature-driven with `Managed` being invoked or not? This is what we need to test here - and also scenario when we do not call `.Managed()` explicitly.

Reply:
[96c7980](https://github.com/opendatahub-io/opendatahub-operator/pull/974/commits/96c79803edbbd36bb6c9d54df42fd08d23fd52f5) reworked!